ContainerFS/Container.cs

```diff
@@ -857,7 +857,7 @@ private bool FileExists(string path, string name)
             {
                 foreach (Tuple<string, long> curr in files)
                 {
-                    if (String.Compare(curr.Item1.ToLower().Trim(), curr.Item1.ToLower().Trim()) == 0)
+                    if (String.Compare(curr.Item1.ToLower().Trim(), name.ToLower().Trim()) == 0)
```

Comment:
Thank you for catching this.  I'll merge and post an update on NuGet later today!